assets/scss/themes/_dark.scss

```diff
@@ -4,20 +4,20 @@
 @mixin dark-theme {
     --theme-name: "dark";
     @include defineColorHSL(--color-primary, $primaryColorDarkH, $primaryColorDarkS, $primaryColorDarkL);
-    @include defineColorHSL(--color-bg, 231, 14%, 10%); // 0%
-    @include defineColorHSL(--color-contrast-lower, 230, 8%, 15%); // 8%
-    @include defineColorHSL(--color-contrast-low, 240, 2%, 25%); // 25%
-    @include defineColorHSL(--color-contrast-medium, 80, 1%, 50%); // 60%
-    @include defineColorHSL(--color-contrast-high, 69, 4%, 68%); // 85%
-    @include defineColorHSL(--color-contrast-higher, 60, 10%, 80%); // 100%
+    @include defineColorHSL(--color-bg, 231, 14%, 10%);
+    @include defineColorHSL(--color-contrast-lower, 230, 12%, 21%);
+    @include defineColorHSL(--color-contrast-low, 240, 2%, 37%);
+    @include defineColorHSL(--color-contrast-medium, 80, 2%, 69%);
+    @include defineColorHSL(--color-contrast-high, 69, 9%, 84%);
+    @include defineColorHSL(--color-contrast-higher, 60, 6%, 90%);
     .theme-icon-light {
         display: none;
     }
     .theme-icon-dark {
         display: inline-block;
     }
     img {
-        filter: brightness(50%);
+        filter: brightness(75%);
```

Comment:
Nice, that's something I currently override in my websites (going for 80% but that's pretty much the same).